spec/models/section_spec.rb

```diff
@@ -1,31 +1,31 @@
 require 'rails_helper'
 
 describe Section do
-  let(:attrs) { FactoryGirl.attributes_for(:section) }
-  let(:course) { double("course") }
-  subject(:section) { Section.new(attrs) }
-  before do
-    allow(course).to receive("id"){ 1 }
-    subject.course_id = course.id
-  end
+  let(:course) { double("Course", :id => 1) }
+
+  subject(:section) { Section.new(
```

Comment:
Can we swap the locations of the `subject` and `let` around so subject is right below the describe and let is below the subject. Its a very minor styling convention thing 😄